README.md

```diff
@@ -6,7 +6,7 @@
 
 **Income and Expense Tracking Standard for Eden on EOS Delegates**
 
-![](https://img.shields.io/github/license/eoscostarica/eden-accounting-standard) ![](https://img.shields.io/badge/code%20style-standard-brightgreen.svg) ![](https://img.shields.io/badge/%E2%9C%93-collaborative_etiquette-brightgreen.svg) ![](https://img.shields.io/twitter/follow/eoscostarica.svg?style=social&logo=twitter) ![](https://img.shields.io/github/forks/eoscostarica/eden-accounting-standard?style=social)
+![](https://img.shields.io/github/license/edenia/eden-spend-explorer) ![](https://img.shields.io/badge/code%20style-standard-brightgreen.svg) ![](https://img.shields.io/badge/%E2%9C%93-collaborative_etiquette-brightgreen.svg) ![](https://img.shields.io/twitter/follow/eoscostarica.svg?style=social&logo=twitter) ![](https://github.com/edenia/eden-spend-explorer/fork)
```

Comment:
change twitter image to edeniaweb3

https://img.shields.io/twitter/follow/edeniaWeb3.svg?style=social&logo=twitter